README.md

```diff
@@ -108,6 +108,57 @@ rm -rf ~/.sedad || true
 ./sedad start
 ```
 
+### Running the Node Yourself Dockerized
+
+For instructions how to run the node yourself as a normal node or a validator in [docker](https://www.docker.com/). 
+
+To pull the `docker` image check [here](#pulling-the-container).
```

Comment:
No link here?

Reply:
Hmm there are links no? One to link to the section and one to the docker website.